src/components/list/EntryList.js

```diff
@@ -23,24 +23,18 @@ const StyledListEntry = styled(ListEntry)`
   }
 `
 
+// TODO: skeleton for unloaded rows and allow arbitrarily scrolling down.
+// See http://bvaughn.github.io/react-virtualized/#/components/InfiniteLoader
+
 const EntryList = forwardRef(
   (
-    {
-      height,
-      width,
-      itemSize,
-      itemCount,
-      locations,
-      onEntryClick,
-      onEntryMouseEnter,
-      onEntryMouseLeave,
-      ...props
-    },
+    { locations, onEntryClick, onEntryMouseEnter, onEntryMouseLeave, ...props },
     ref,
   ) => {
     const renderRow = ({ index, style }) => {
       const location = locations[index]
-      let row = null
+
+      let row
```

Comment:
any reason we're no longer using null here?

Reply:
Yes, it's an if/else clause now, so always will be filled in.